src/main/python/linuxband/gui/chord_entries.py

```diff
@@ -164,7 +171,7 @@ def __get_chord_names(self):
 
         chord_names = []
         for c in base_ext:
-            for k, v in chordlist.iteritems(): #@UnusedVariable
+            for k, v in chordlist.iteritems():
```

Comment:
I used the #@UnusedVariable to suppress a warning in Eclipse. What about following this advice: https://stackoverflow.com/questions/13944234/how-to-suppress-unused-variable-warnings-in-eclipse-pydev/13944314#13944314

This would work for me:
for k, _ in chordlist.iteritems():

Reply:
Do you have any objections against using `cordlist.iterkeys()`? This seems to be available in Python2.

Reply:
`cordlist.iterkeys()` sounds great!